packages/forms/src/Components/Concerns/CanBeValidated.php

```diff
@@ -627,4 +638,28 @@ public function multiFieldComparisonRule(string $rule, array | string | Closure
 
         return $this;
     }
+
+    public function multiFieldValueComparisonRule(string $rule, string | Closure $statePath, mixed $stateValues, bool $isStatePathAbsolute = false): static
```

Comment:
Shouldn't this rather be `protected`, @danharrin?

Reply:
Similar methods are public. Also, it does work when used on its own, so I don't see a problem